server/app.py

```diff
@@ -27,18 +27,8 @@
 # Register flask_jwt_extended with Flask
 jwt = JWTManager(app)
 
-
+app.register_blueprint(search_views)
 app.register_blueprint(home_handler)
 app.register_blueprint(ping_handler)
 app.register_blueprint(auth_views)
-
-@jwt.user_loader_error_loader
-def custom_user_loader_error(identity):
-    """
-    Identity is set to be the username
-    Make a more user friendly error
-    """
-    return  {
-        'status': 404,
-        "message": "User with username %s not found" % (identity)
-    }
+app.register_blueprint(search_views)
```

Comment:
I think you registered this blueprint twice

Reply:
Okay, I'll remove that